site/docs/v1/tech/apis/applications.adoc

```diff
@@ -162,6 +163,125 @@ include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_application-response-body.adoc[]
 
+== Search for Applications
+
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+This API is used to search for Applications and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Applications using the given search and sort criteria
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/application/search``?name=\{name\}&active=\{active\}&tenantId=\{tenantId\}&orderBy={orderBy}&startRow={startRow}``#
+--
+
+==== Request Parameters
+
+[field]#name# [type]#[String]# [optional]#Optional#::
+The string to search for in the Application name. This can contain wildcards using the asterisk or percent characters (* or %). This search parameter is case-insensitive.
+
+[field]#active# [type]#[Boolean]# [optional]#Optional#::
+Filters on the Application's `active` status. Can be omitted or set to `null` to retrieve both active and inactive Applications.
+
+[field]#tenantId# [type]#[UUID]# [optional]#Optional#::
+Restricts the results to Applications belonging to the given Tenant. This parameter will be overridden if the request contains an `X-FusionAuth-TenantId` header, or if the supplied API key is scoped to a specific Tenant.
+
+[field]#numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `insertInstant DESC`#::
+The database column to order the search results on plus the order direction.
++
+The columns you can use for this are:
++
+ * `id` - the UUID of the Application
+ * `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Application was created
+ * `name` - the Application name
+ * `tenant` - the name of the Tenant in which the Application belongs
++
+For example, to order the results by the insert instant in a descending order, the value would be provided as `insertInstant DESC`. The final string is optional can be set to `ASC` or `DESC`. If omitted, the default sort order is `ASC`.
```

Comment:
What happens if I put something other than `ASC` or `DESC`?

Reply:
```suggestion
The order suffix  is optional. Possible values of the order suffix are `ASC` or `DESC`. If omitted, the default sort order is `ASC`.
+
For example, to order the results by the insert instant in a descending order, use `insertInstant DESC`.
```

Reply:
I moved this around because I wanted to explain each part of the string before showing an example.

Also, I don't love the term `order suffix` because it feels fancy, but couldn't think of a better term.

Reply:
Maybe `order direction`? I think that most developers will be familiar.

I just confirmed that specifying something other than `ASC` or `DESC` results in a validation error.

```
The [search.orderBy] property contains an invalid column name. The invalid column name is [insertInstant GAR]. The supported values are [id, name, insertInstant].
```

Does that validation message need to be expanded to mention valid sort directions? That would be a change to existing APIs as well. I'm not sure whether changing a validation error message is considered a breaking change.

Reply:
I think order direction is fine.

I don't know about whether modifying an API error message is a breaking change. I think `no` because we don't document them anywhere, but am not sure.

However, that error message isn't great, because it doesn't tell you that `GAR` is the issue. I'll leave it up to you whether you want to change it.